Chess/common_types.h

```diff
@@ -0,0 +1,61 @@
+#ifndef COMMON_TYPES_H
+#define COMMON_TYPES_H
+
+#include <QObject>
+#include <QString>
+
+
+class Figure;
+class Desk;
+class Command;
+
+
+enum FigType{EMPTY/*base class*/,
+             KING,
+             QUEEN,
+             ROOK,
+             BISHOP,
+             KNIGHT,
+             PAWN};
+
+
+enum FigColor{NONE/*base class*/,
+              WHITE,
+              BLACK};
+
+
+struct Cell {
```

Comment:
Why do you use struct? better use Class and access data via get_something set_something methods